tools/RestfulApiWebService/OperationOnDB/Global.asax.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using System.Web.Routing;
+using System.Web.Security;
+using System.Web.SessionState;
+using System.Web.Http;
+
+namespace OperationOnDB
+{
+    public class Global : HttpApplication
+    {
+        void Application_Start(object sender, EventArgs e)
+        {
+            // 在应用程序启动时运行的代码
```

Comment:
remove this comment? which is produced automatically

---
In reply to: [129222616](https://github.com/dotnet/docfx/pull/1906#discussion_r129222616) [](ancestors = 129222616)